### ISSUE TITLE
Moving launcher from one panel to another leaves a gap in the first launcher

### ISSUE THREAD

**Issue by CottonEaster** [linuxmint/Cinnamon#7791]
```
 Cinnamon version (cinnamon --version) 3.8.8.
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Mint 19
 * Graphics hardware *and* driver used: Intel UHD Graphics 620; 3.0 Mesa 18.0.5
 * 32 or 64 bit: 64
 ```

**Issue**

**Steps to reproduce**

Have two panels.
Use the Mint Menu - via, I think, either panel - to select an app and pin it to the (well, a) panel.
Move the resulting launcher from the panel on which it appeared (for that might not be the one you want it on) to the other panel.
Observe - if the problem proves reproducible - that a gap remains on the panel where the launcher used to be. I found that after a while - because of mouse-over? - the hole left by the departing icon disappeared.

**Expected behaviour**

**Other information**

**Comment by leigh123linux** [comment]
Please complete the template properly and completely, maybe include a screenshot.

**Comment by CottonEaster** [comment]
@leigh123linux 
Sorry, but what's wrong with how I completed the template? I completed all its fields, so far as I can see.

I tried to take a video of the phenomenon, but I find that, at present, that I cannot recreate the problem. Seemingly certain conditions are required. I can add that the gap persisted  - when the problem did occur - for some ten seconds or more. I will try to get a screenshot if the problem occurs again (though mostly I do not use two panels). Perhaps compare #2139.
